src/flows/components/ExportTaskOverlay/ExportTaskOverlay.tsx

```diff
@@ -0,0 +1,344 @@
+// Libraries
+import React, {ChangeEvent, FC, useEffect, useState} from 'react'
```

Comment:
Totally hear you on that. I felt the same way. My concern was updating the parent state from the child felt wrong, but if we were to set the state directly within the child we would've lost context (and since this exists outside of `context` I wasn't sure how to get the two to work together)